### DISCUSSION TITLE
Exclude Specifics Statistics from Update

### DISCUSSION THREAD

**Discussion by mildmanneredjon** [olahallengren/sql-server-maintenance-solution#569] [Questions & Answers]
Hi, is it possible to exclude a specific column statistic from been updated? I have a varbinary(max) column that I do not want the stats updated on but can't see that you can name a statistic to be excluded like you can with indexes. Thanks.

**Comment by dbgresham** [comment]
I do not have an answer but just wanted to add that I have the same question.  I have 2 columns that fail every time because tempdb is too small.  It is sufficient for everything else so managers do not want to add a lot of space for a task that is done once a week.  For now I am thinking of excluding the tables with these columns from the standard job, then running a separate step for each of them with @UpdateStatistics='ALL'.@StatisticsSample='20' followed by another with @UpdateStatistics='INDEX',@StatisticsSample='100'.  The tempdb is large enough to gather column statistics if we sample, but they want the index statistics to look at all the dat.

I would just exclude the 2 column stats from the standard job and do them separately if there is a way to do that.  Any other suggestions are appreciated.  Thanks.

**Reply by mildmanneredjon** [reply]
Hi. Yes that's exactly what I did. Excluded tables from stats update, then created a separate agent step to manually update the stats in the specific tables excluding the large string columns. It works and so far only have 2 tables this is affecting so an easy work around
